lib/systemd/system-sleep/pop-default-settings_bluetooth-suspend

```diff
@@ -2,15 +2,19 @@
 
 set -e
 
-#TODO: save rfkill state?
 case "$2" in
     suspend | hybrid-sleep)
         case "$1" in
             pre)
-				rfkill block bluetooth
+                if [ $(cat /var/lib/systemd/rfkill/*bluetooth) -eq 1 ]; then
+                    > /home/$USER/blocked.bt
+                elif [ $(cat /var/lib/systemd/rfkill/*bluetooth) -eq 0 ]; then
```

Comment:
This has the same issue as above... perhaps you should introduced a variable:

```
BT_BLOCKED=$(cat /var/lib/systemd/rfkill/*bluetooth | sort -r | head -n 1)
case $BT_BLOCKED in
0)
[ ! -f /run/bluetooth.blocked ] && rfkill unblock bluetooth
rm /run/bluetooth.blocked
;;
1)
> /run/bluetooth.blocked
;;
```

Using a `case` statement would also allow us to handle the case where there is no bluetooth device.

Reply:
This works.